tests/snuba/api/endpoints/test_organization_events_span_metrics.py

```diff
@@ -120,6 +120,53 @@ def test_count_if(self):
         assert data[0][fieldRelease2] == 1
         assert meta["dataset"] == "spansMetrics"
 
+    def test_division_if(self):
+        self.store_span_metric(
+            1,
+            internal_metric=constants.SPAN_METRICS_MAP["mobile.slow_frames"],
+            timestamp=self.three_days_ago,
+            tags={"release": "1.0.0"},
+        )
```

Comment:
These look like they'd pass but that's just because of how the data is being mocked and where the mock data is set up. There's another example of slow frames tests here where the first arg is actually passed as an object which is consistent for the gauge type, I'd suggest changing these to match that type to make sure it's working the way you want for these metrics, e.g.

```
        self.store_span_metric(
            {
                "min": 1,
                "max": 1,
                "sum": 1,
                "count": 1,
                "last": 1,
            },
            entity="metrics_gauges",
            metric="mobile.slow_frames",
            timestamp=self.six_min_ago,
            tags={"release": "foo"},
        )
```

Total frames might be the only one that needs to have different values for min/max/sum/count/last when entered